src/main/java/ch/heigvd/pro/b04/questions/QuestionController.java

```diff
@@ -116,6 +118,14 @@ public QuestionController(QuestionRepository repository,
         .build()));
   }
 
+  private void verifyAndInsertIndexInPoll(ServerPoll poll, double index)
+      throws IndexInPollAlreadyExist {
+    if (poll.getAllIndexs().contains(index)) {
+      throw new IndexInPollAlreadyExist();
+    }
+    poll.getAllIndexs().add(index);
```

Comment:
This writes the index to a Java field, rather than checking with data from the database.